justfile

```diff
@@ -122,6 +122,9 @@ logs-cloudbeat:
 deploy-cloudformation:
   cd deploy/cloudformation && go run .
 
+create-cnvm-stack-tags region stack_name tags='Key=division,Value=engineering Key=org,Value=security Key=team,Value=cloud-security-posture':
```

Comment:
Can you put the default common tags into a const instead?

Reply:
I think it's missing the `project` tag. based on [this](https://docs.google.com/document/d/1FdkJ5EM1tSAqVRvZvmtDSXbBrzS8f7Bw2vENTFes1ek/edit#heading=h.36dv1gu8gyr7)

Reply:
justfile is updated